UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPickerViewController.swift

```diff
@@ -384,8 +384,7 @@ extension UnsplashPhotoPickerViewController: PagedDataSourceDelegate {
     }
 
     func dataSource(_ dataSource: PagedDataSource, didFetch items: [UnsplashPhoto]) {
-        guard items.count > 0 else { return }
```

Comment:
@JoshRobertsDev This line was added by @zsbee because of this:

https://github.com/lukilabs/unsplash-photopicker-ios/commit/8d8bfd67428e96424d8390a459add7b1bcca090a

Can you please check if the issue is not reintroduced?

Reply:
Looks like both can be fixed by just moving the call of this. I didn't realise at first there was a difference between datasource.items and items in from the function param. That function param is new stuff that's been paged so returning early still fixes the flicker, just needs to be under the handling of the empty state so it's handled first